### ISSUE TITLE
Is there a string replace?

### ISSUE THREAD

**Issue by anuradharajan** [WorkMaze/JUST.net#216]
Input
{
"test":"12345xxxx"
}

Expected output (Replace xxxx with empty string)
{
"result":"12345"
}

**Comment by Courela** [comment]
No, but probably it can be accomplished using existing functions.
